tests/models/ddl_mysql.sql

```diff
@@ -0,0 +1,6 @@
+-- atlas:pos user_account[type=table] tests/models/models.py:10
+-- atlas:pos address[type=table] tests/models/models.py:23
+CREATE TABLE user_account (id INTEGER NOT NULL AUTO_INCREMENT, name VARCHAR(30) NOT NULL, fullname VARCHAR(30), PRIMARY KEY (id));
```

Comment:
File-directives should be detached from the statements with \n\n
```suggestion
-- atlas:pos address[type=table] tests/models/models.py:23

CREATE TABLE user_account (id INTEGER NOT NULL AUTO_INCREMENT, name VARCHAR(30) NOT NULL, fullname VARCHAR(30), PRIMARY KEY (id));
```

Reply:
Ref: https://atlasgo.io/atlas-schema/sql#file-level-directives